README.md

```diff
@@ -784,7 +784,8 @@ A collection of awesome things regarding React ecosystem.
 * [VictoryJS - A collection of composable React components for building interactive data visualizations](https://github.com/FormidableLabs/victory)
 * [Recharts - A charting library built on D3 with an awesome declarative API](https://github.com/recharts/recharts)
 
-
+#### ReactJS Interview Questions and Answers
```

Comment:
can you change this to `Job Interview Tools`, add it to the table of contents, maybe find some other sources? :)

Reply:
Thanks for review. Created a new MR https://github.com/enaqx/awesome-react/pull/836 . So closing this MR.